### PR TITLE
Async conf

### PR DESCRIPTION
**Pull request by archlichking**
add `--sync_browsers` param for magellan

@Maciek416 @geekdave

### CODE REVIEW

File: lib/rewrite_config.js [hunk 1]
```diff
@@ -8,9 +8,15 @@ var CHROMEDRIVER_LOCATION = "./node_modules/chromedriver/lib/chromedriver/chrome
 var PHANTOMJS_LOCATION = "./node_modules/phantomjs/bin/phantomjs";
 // throws file read/write exceptions, JSON parse exceptions
 module.exports = function (sourceConfigPath, tempAssetPath, options) {
-
   var conf = require(path.resolve(sourceConfigPath));
 
+  if (options.syncBrowsers) {
+    if (!conf.test_settings.default.globals) {
+      conf.test_settings.default.globals = {};
```
Comment (Maciek416) [inline lib/rewrite_config.js:15]: is this deep enough or should we check the whole path?

**Comment by Maciek416** [comment]
aside from one comment, 👍 LGTM

File: lib/rewrite_config.js (hunk 1 above)
Comment (archlichking) [reply lib/rewrite_config.js:15]: `conf.test_settings.default` is deep enough

**Comment by Maciek416** [comment]
👍

**Pull request merged**